### ISSUE TITLE
[CoE Starter Kit - BUG] Bots page State column showing published for unpublished bots

### ISSUE THREAD

**Issue by morismm99** [microsoft/coe-starter-kit#8794]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Custom Copilots which have not been published (just saved) show up as Published for the State column of the Bots page of the CoE Admin Dashboard PBI. Looks like we are grabbing the body/componentstate, which seems to incorrectly show Published for unpublished Bots. I have confirmed the raw Chatbot table, component state column also shows published. However, the bot is definitely not published. 
![image](https://github.com/user-attachments/assets/3873f060-8230-455e-b784-0da1ac6214be)
![image](https://github.com/user-attachments/assets/4e753b6f-c680-4fef-8039-f3e1545cd02a)
This is causing confusing for power platform admins, as some Copilots show up under users who are not licensed to publish custom bots, so they have to do extra work to make sure a user has not been accidentally licensed if not approved to create custom copilots. For example, users with a Copilot for M365 license can access Copilot Studio to create plugins for 1st party copilots, and they can still create custom copilots but not published...However, when they attempt to create the custom copilot, these are saved (not published) and show up as published in the reporting.

### Expected Behavior

We would expect the copilot state to show as "saved" instead of Published, or something to show it is not published. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.33

### What app or flow are you having the issue with?

Admin | Sync Template v4 (PVA)

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

1. Issue is apparent on the Admin CoE Dashboard PBI report
2. Confirmed the Admin | Sync Template v4 (PVA) flow is grabbing the correct column, so we may need to reach out to PG to ask if we could correctly capture the bot's true state.

**Comment by Jenefer-Monroe** [comment]
Yes I just looked at the product's tables and it does appear they are always showing as status published there, so this is something that will need to be taken to product support. 
I'm marking as a product issue, and can reach out internally to try and get traction. But you posting support tickets will make it more likely to get fixed by the product team, so please do.

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation.

**Comment by mohamrizwa** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to a new issue. We’re happy to help anytime. Thanks again for your engagement!
